### PR TITLE
Bad JS path in view can trigger InvalidCrossOriginRequest

### PR DESCRIPTION
**Pull request by bobmazanec**
Several hours over several days I'll never get back...

### CODE REVIEW

File: rails/bad-paths-can-trigger-invalid_cross_origin_request.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+to
+
+    app/assets/javascripts/scorebook/create_a_class.js
+
+It seemed to work fine in `development`, but my new Capybara `feature`
+spec triggered
+
+    Failure/Error: Unable to find matching line from backtrace
+    ActionController::InvalidCrossOriginRequest:
+      Security warning: an embedded <script> tag on another site requested protected JavaScript. If you know what you're doing, go ahead and disable forgery protection on this action to permit cross-origin JavaScript embedding.
```
Comment (JoelQ) [inline rails/bad-paths-can-trigger-invalid_cross_origin_request.md]: Can we break this up into multiple lines to there is no horizontal scrolling to read the error message?

**Comment by bobmazanec** [comment]
Good point; how now?

File: rails/bad-paths-can-trigger-invalid_cross_origin_request.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      disable forgery protection on this action to permit cross-origin
+      JavaScript embedding.
+
+until I changed the view's
+
+    <%= javascript_include_tag 'create_a_class.js' %>
+
+to
+
+    <%= javascript_include_tag 'scorebook/create_a_class.js' %>
```
Comment (JoelQ) [inline rails/bad-paths-can-trigger-invalid_cross_origin_request.md]: Thoughts on using the backticks syntax to get erb syntax highlighting?

```erb
<%= javascript_include_tag 'scorebook/create_a_class.js' %>
```

File: rails/bad-paths-can-trigger-invalid_cross_origin_request.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Bad JS paths can trigger `InvalidCrossOriginRequest`
+
+Rails v4.1.6
+
+In a controller using [`protect_from_forgery`](http://guides.rubyonrails.org/security.html#csrf-countermeasures),
+moving (but not otherwise renaming) a JS file referenced by that
+controller's view(s) without updating the view(s) can trigger an
```
Comment (JoelQ) [inline rails/bad-paths-can-trigger-invalid_cross_origin_request.md]: Could we simplify this paragraph by just saying this error can be triggered by including an incorrect script path?

**Comment by bobmazanec** [comment]
Done?

File: rails/bad-paths-can-trigger-invalid_cross_origin_request.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+# Bad JS paths can trigger `InvalidCrossOriginRequest`
```
Comment (thorncp) [inline rails/bad-paths-can-trigger-invalid_cross_origin_request.md]: Maybe pedantic, but can you shorten this title so it doesn't wrap to a second line? The exception class is already called out in the test failure, so I don't think it's required in the title.

Comment (bcjordan) [inline rails/bad-paths-can-trigger-invalid_cross_origin_request.md]: slightly shorter if it helps:

> Bad JS Path Triggers [...]

**Comment by stefannibrasil** [comment]
This repo is no longer active and it's being archived. The contents will still be available as read-only. Thank you for your contributions!

**Pull request closed without merging**